common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/store/util/LwM2MClientSerDes.java

```diff
@@ -111,14 +110,7 @@ public static byte[] serialize(LwM2mClient client) {
         o.addProperty("defaultObjectIDVer", client.getDefaultObjectIDVer().toString());
 
         if (client.getRegistration() != null) {
-            String registrationAddress = client.getRegistration().getAddress().toString();
             JsonNode registrationNode = registrationSerDes.jSerialize(client.getRegistration());
```

Comment:
Security Node: PSK?

Reply:
I think PSK not related to these changes, but ok, I will deploy it to the tbqa and we will check.